### PR TITLE
built new static page

### PR DESCRIPTION
**Pull request by marwin89**
Here is a new static page for specification. 
built with lazyfox template (open source, html5, bootstrap3).
Feel free to edit the page. its just a first try.
I used rakuris txt-documentation.

format-static-page

### CODE REVIEW

File: docs/.gitignore [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Windows image file caches
```
Comment (Baklap4) [inline docs/.gitignore:1]: Please move this file to the root of the solution

File: docs/.gitattributes [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Auto detect text files and perform LF normalization
```
Comment (Baklap4) [inline docs/.gitattributes:1]: Please move this file to the root of the solution

File: docs/.gitattributes [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+* text=auto
```
Comment (Baklap4) [inline docs/.gitattributes:2]: As specified in the editorconfig line-endings are always lf for all text files

**Review by Baklap4 — CHANGES_REQUESTED** [review]
Thanks a lot for the PR 👍 
A couple of changes for now like moving some files throughout the repository

Then for the libraries (bootstrap, jquery, css, fonts), i'd advise to use a cdn for those and then don't include these minified files so we host only the custom css/js :)

**Comment by basisbit** [comment]
> Then for the libraries (bootstrap, jquery, css, fonts), i'd advise to use a cdn for those and then don't include these minified files so we host only the custom css/js :)

I disagree, please keep the minified files in this here repository where the license allows it. This is a requirement to comply with GDPR/CCPA/LGPD. Thanks.

File: docs/.gitignore (hunk 1 above)
Comment (marwin89) [reply docs/.gitignore]: done

File: docs/.gitattributes (hunk 3 above)
Comment (marwin89) [reply docs/.gitattributes]: updated gitattributes

**Review by Baklap4 — APPROVED** [review]

**Review by Baklap4 — APPROVED** [review]

**Pull request merged**